Makefile

```diff
@@ -0,0 +1,11 @@
+test:
+	pipenv run detox
+
+ci:
+	pipenv run py.test --cov=stripe -n 8
```

Comment:
The preferred name is just "pytest" (also above).

Reply:
👍 Thanks! Will fix.